src/index.coffee

```diff
@@ -2,7 +2,7 @@ module.exports =
   rules:
     'no-undef': require './rules/no-undef'
     'jsx-no-undef': require './rules/jsx-no-undef'
-    'no-unused-vars': require './rules/no-unused-vars'
+    'no-unused-vars': require './rules/no-unused-vars-typescript'
```

Comment:
It looks like (as I'd expect) the typescript-eslint version of `no-unused-vars` works just fine against JS (so we can effectively replace the old `no-unused-vars` with this one)